changes/ce/fix-10107.zh.md

```diff
@@ -0,0 +1,6 @@
+对于对 `Bridges API` 的操作，如果 `bridge-id` 未知，我们现在返回 `404`，而不是
+`400`。此外，还修复了一个错误，即如果这是一个节点的操作，会导致崩溃。此外，我们
+现在也检查集群操作 `start`，如果给定的桥被启用。受影响的端点。
+ * [cluster] `/bridges/:id/:operation`,
+ * [node] `/nodes/:node/bridges/:id/:operation`,
+其中 `operation` 是 `[start|stop|restart]` 之一。
```

Comment:
```suggestion
修复 `Bridges` 相关的 HTTP API 错误：
1. 当请求的 `bridge-id` 未知时，现在返回 `404` 而不是 `400`。
2. 修复对 `nodes/:node/bridges/:id/:operation` 相关的调用，存在的崩溃错误。
3. 当执行 `start` 操作时会检查该桥接是否已经被启用，否则会返回错误。
受影响的路径有：
 * [cluster] `/bridges/:id/:operation`,
 * [node] `/nodes/:node/bridges/:id/:operation`,
其中 `operation` 是 `[start|stop|restart]` 之一。
```

Reply:
@lafirest this one or yours?

Reply:
this one is also fine, you can use this one.